recipe/meta.yaml

```diff
@@ -12,20 +11,22 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: True # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - pip
-    - pytest-runner >=2.11.1
-    - python >=3.7,<3.12
+    - wheel
+    - python
+    - setuptools
+    - pytest-runner 6.0.0
   run:
-    - Faker >=10
+    - python
+    - faker >=10
     - numpy >=1.20.0,<2
     - pandas >=1.1.3
     - psutil >=5.7,<6
-    - python >=3.7,<3.12
     - scipy >=1.5.4,<2
     - scikit-learn >=0.24,<2
```

Comment:
Compatibility for py<310 and py==311 is much more complicated https://github.com/sdv-dev/RDT/blob/8e8a649fa2f3e917d12fed3c36b0d32c1a9ed561/setup.py#L18-L26

Reply:
Those are specified on the basis of available version on `conda-forge`, and not any actual technical reasons. I verified this by running the full test suite with the oldest versions available on all Python versions.